udata/core/dataset/models.py

```diff
@@ -68,6 +68,8 @@
 
 
 class License(db.Document):
+    # We need to declare id explicitly since we do not use the default
+    # value set by Mongo.
```

Comment:
I don't know if we need to explain here a documented and common feature of MongoEngine/MongoDB.
But, right now, if it helps, let's keep it 👍